integration_tests/integration/homepage/homepage.cy.js

```diff
@@ -15,9 +15,15 @@ context('Homepage', () => {
   })
 
   describe('Tasks', () => {
-    it('should return a 401 if there are no roles present', () => {
+    it('should only show help page if there are no roles present', () => {
```

Comment:
I didn't want to attach roles to the help page because then you have to remember to add any new roles. Currently Auth _does_ show a 401 type page if the user tries to access Restricted Patient without any roles, but that's up to Auth. As far as this application is concerned anyone can see the help page.